### DISCUSSION TITLE
Query on CPU Throttling, Simulated Throttling and DevTools Throttling

### DISCUSSION THREAD

**Discussion by daphneyyt** [GoogleChrome/lighthouse#15762] [Q&A]
Hi Google Lighthouse team, we have a few questions regarding CPU throttling.

Context: we are using Google Lighthouse in the Google Chrome extension mainly for performance checks).

**Question on CPU throttling**
1) We would like to know if CPU throttling is both present in simulated and devtools throttling.

2) Can we disable CPU throttling?

**Understanding the difference between Simulated Throttling and DevTools Throttling**
3a) Additionally, would like to understand the general difference between 'simulated throttling' and 'devtools throttling'. We read up on some literature regarding the two types of settings, but it is still unclear to us.

3b) Also, when is the appropriate situation to use either one of these throttling settings?

**Difference in Desktop and Mobile loading speed**
4) Lastly, as a general question, we would like to understand why mobile loading speed is slower than desktop loading speed?

Thank you.

**Comment by adamraine** [comment]
> We would like to know if CPU throttling is both present in simulated and devtools throttling.

It is present in both when using the default mobile config.

> Can we disable CPU throttling?

Yes, the following config will not have any CPU throttling

```js
const config = {
  extends: 'lighthouse:default',
  settings: {
    throttling: {
      cpuSlowdownMultiplier: 1,
    }
  }
}
```

> 3a) Additionally, would like to understand the general difference between 'simulated throttling' and 'devtools throttling'. We read up on some literature regarding the two types of settings, but it is still unclear to us.

The following section in our throttling doc explains the difference: https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md#types-of-cpu-throttling

> 3b) Also, when is the appropriate situation to use either one of these throttling settings?

See the doc I linked in my answer to 3a. As the doc explains for simulated CPU throttling:

> The TLDR: while it's fairly accurate for most circumstances, it suffers from edge cases and a deep investigation to performance should use DevTools CPU throttling tools.

> 4) Lastly, as a general question, we would like to understand why mobile loading speed is slower than desktop loading speed?

In general, mobile devices have slower page loads because of slower device hardware (CPU) and slower network connections (3G/4G vs wifi)
